### ISSUE TITLE
python collect.py后输出储存失败

### ISSUE THREAD

**Issue by yelfen** [tensorfly-gpu/aichess#5]
已加载初始模型
F:\aichess-main\pytorch_net.py:75: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  policy = F.log_softmax(policy)
D:\Anaconda\envs\chess\lib\site-packages\torch\nn\functional.py:1949: UserWarning: nn.functional.tanh is deprecated. Use torch.tanh instead.
  warnings.warn("nn.functional.tanh is deprecated. Use torch.tanh instead.")
走一步要花:  5.746572971343994
走一步要花:  5.981132745742798
F:\aichess-main\zip_array.py:51: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  return np.array(zip_res)
存储失败

**Comment by yelfen** [comment]
关键是都是warning，也不说知道是哪里错

**Comment by tensorfly-gpu** [comment]
刚刚更新了代码，你现在重新fork一下，在训练应该就没有问题了。

**Comment by tensorfly-gpu** [comment]
> 关键是都是warning，也不说知道是哪里错

刚刚更新了代码，你现在重新fork一下，在训练应该就没有问题了。

**Comment by yelfen** [comment]
> > 关键是都是warning，也不说知道是哪里错
> 
> 刚刚更新了代码，你现在重新fork一下，在训练应该就没有问题了。

解决了，闪电回复
